        <html><head></head><body>
        <h1>Report for aspell-es on toolchain
            <a href="index.html">android</a>
        </h1>
        <p>Return code: 1</p>
        <p>Time to run: 00:00:30</p>
        <p>Log output:</p>
        <code><pre><docker run --rm -v /tuscan_data --volumes-from tuscan_data -v /sources --volumes-from sources -v /var/cache/pacman/pkg --volumes-from pkg_cache_android -v logs:/logs make_package_container --sources-directory /sources --shared-directory /tuscan_data aspell-es
:: Synchronizing package databases...
downloading repo.db...
downloading core.db...
downloading extra.db...
downloading community.db...
Found 2313 packages in cache
===> sudo -u tuscan makepkg --nobuild --syncdeps --skipinteg --skippgpcheck --skipchecksums --noconfirm --nocolor --log --noprogressbar --nocheck
==> Making package: aspell-es 1.11-6 (Sat Nov 14 00:18:07 UTC 2015)
==> Checking runtime dependencies...
==> Installing missing dependencies...
resolving dependencies...
looking for conflicting packages...

Packages (1) aspell-0.60.6.1-3

Total Installed Size:  3.00 MiB

:: Proceed with installation? [Y/n] 
checking keyring...
checking package integrity...
loading package files...
checking for file conflicts...
checking available disk space...
installing aspell...
==> aspell comes with no default dictionary
Optional dependencies for aspell
    perl: to import old dictionaries [installed]
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Downloading aspell6-es-1.11-2.tar.bz2...
==> WARNING: Skipping all source file integrity checks.
==> Extracting sources...
  -> Extracting aspell6-es-1.11-2.tar.bz2 with bsdtar
==> Sources are ready.
Copied source directory to /sources/aspell-es
Traceback (most recent call last):
  File "/build/make_package.py", line 215, in <module>
    main()
  File "/build/make_package.py", line 212, in main
    result = copy_and_build(args)
  File "/build/make_package.py", line 124, in copy_and_build
    pkgbuild = f.read().splitlines()
  File "/usr/lib/python3.5/codecs.py", line 321, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf3 in position 140: invalid continuation byte></pre></code>
        </body></html>
